modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -225,7 +225,7 @@ export default function ActionBuilder({
 
 					<a
 						className="alert-link"
-						href="https://learn.liferay.com/dxp/latest/en/building-applications/objects/creating-and-managing-objects/expression-builder-validations-reference.html"
+						href="https://learn.liferay.com/en/w/dxp/building-applications/objects/creating-and-managing-objects/actions"
```

Comment:
Here the text says 'create-conditions-and-predefined-values-using-expressions', but the link points to the Actions documentation, so it creates another inconsistency. We should change the text here, or leave the link as it was.